### ISSUE TITLE
Nswag build

### ISSUE THREAD

**Issue by satishviswanathan** [RicoSuter/NSwag#2233]
I have the latest source code downloaded and trying to build the solution but I'm getting some errors. If you have any guidelines around this , then would you be able to share them.

**Comment by RicoSuter** [comment]
See https://github.com/RicoSuter/NSwag/wiki/SDK-Development

Please ping me when there are open questions

**Comment by satishviswanathan** [comment]
I modified NSwag.ConsoleCore project and set the arguments to point to the location where I have my .nswag file.

When I build the solution I'm getting the following error. Not sure what i'm missing.

Severity	Code	Description	Project	File	Line	Suppression State
Error		The command "rmdir "../../../../src/NSwagStudio/bin/Debug/Win" /Q /S nonemptydir
rmdir "../../../../src/NSwagStudio/bin/Debug/NetCore10" /Q /S nonemptydir
rmdir "../../../../src/NSwagStudio/bin/Debug/NetCore11" /Q /S nonemptydir
rmdir "../../../../src/NSwagStudio/bin/Debug/NetCore20" /Q /S nonemptydir
rmdir "../../../../src/NSwagStudio/bin/Debug/NetCore21" /Q /S nonemptydir
rmdir "../../../../src/NSwagStudio/bin/Debug/NetCore22" /Q /S nonemptydir

dotnet restore "../../../../src/NSwag.ConsoleCore" --no-cache

dotnet publish "../../../../src/NSwag.ConsoleCore" -c Debug -f "netcoreapp1.0" -o "bin/Debug/netcoreapp1.0/Publish" /property:Platform=AnyCPU
dotnet publish "../../../../src/NSwag.ConsoleCore" -c Debug -f "netcoreapp1.1" -o "bin/Debug/netcoreapp1.1/Publish" /property:Platform=AnyCPU
dotnet publish "../../../../src/NSwag.ConsoleCore" -c Debug -f "netcoreapp2.0" -o "bin/Debug/netcoreapp2.0/Publish" /property:Platform=AnyCPU
dotnet publish "../../../../src/NSwag.ConsoleCore" -c Debug -f "netcoreapp2.1" -o "bin/Debug/netcoreapp2.1/Publish" /property:Platform=AnyCPU
dotnet publish "../../../../src/NSwag.ConsoleCore" -c Debug -f "netcoreapp2.2" -o "bin/Debug/netcoreapp2.2/Publish" /property:Platform=AnyCPU

xcopy "../../../../src/NSwag.Console.x86/bin/Debug/net461" "../../../../src/NSwagStudio/bin/Debug/Win" /E /I /y
xcopy "../../../../src/NSwag.Console/bin/Debug/net461" "../../../../src/NSwagStudio/bin/Debug/Win" /E /I /y

xcopy "../../../../src/NSwag.ConsoleCore/bin/Debug/netcoreapp1.0/publish" "../../../../src/NSwagStudio/bin/Debug/NetCore10" /E /I /y
xcopy "../../../../src/NSwag.ConsoleCore/bin/Debug/netcoreapp1.1/publish" "../../../../src/NSwagStudio/bin/Debug/NetCore11" /E /I /y
xcopy "../../../../src/NSwag.ConsoleCore/bin/Debug/netcoreapp2.0/publish" "../../../../src/NSwagStudio/bin/Debug/NetCore20" /E /I /y
xcopy "../../../../src/NSwag.ConsoleCore/bin/Debug/netcoreapp2.1/publish" "../../../../src/NSwagStudio/bin/Debug/NetCore21" /E /I /y
xcopy "../../../../src/NSwag.ConsoleCore/bin/Debug/netcoreapp2.2/publish" "../../../../src/NSwagStudio/bin/Debug/NetCore22" /E /I /y" exited with code 4.	NSwagStudio.Installer	C:\Program Files (x86)\MSBuild\Microsoft\WiX\v3.x\wix2010.targets	779

**Comment by RicoSuter** [comment]
If you only want to test NSwag.ConsoleCore then use NSwag.Min.sln
way easier to build :-)

**Comment by satishviswanathan** [comment]
With the changes I mentioned, I was trying to prepare the msi using the NSwagStudio.Installer proj. But when I build that project I'm getting this error.

**Comment by RicoSuter** [comment]
Did you install the wix sdk?

**Comment by satishviswanathan** [comment]
Yes after installing wix the installer project got loaded and when I build the installer project I'm getting this error.

**Comment by RicoSuter** [comment]
Do you have .net core 2.2 installed?

**Comment by satishviswanathan** [comment]
Yes I do have .Net core 2.2 installed

**Comment by RicoSuter** [comment]
Try running build/01_build.bat in a vs cmd line shell, hopefully you have more info there..

**Comment by RicoSuter** [comment]
Ah it’s really bad supporting all these frameworks.. 🙂

**Comment by satishviswanathan** [comment]
yeah totally agree :)

**Comment by satishviswanathan** [comment]
I ran sharing the log file. 

[nswag_log.txt](https://github.com/RicoSuter/NSwag/files/3292317/nswag_log.txt)

**Comment by satishviswanathan** [comment]
My Bad, I didn't run the bat file from VS command prompt. Sharing the correct log.

[nswag_log.txt](https://github.com/RicoSuter/NSwag/files/3292359/nswag_log.txt)
